app/views/api/v1/quotes/show.json.jbuilder

```diff
@@ -1,5 +1,5 @@
 json.quote do
-  json.quoted_person @quote.person.full_name
+  json.quoted_person @quote.person.full_name || @quote.person.slack_name
```

Comment:
Seems like this appears in a few places. Would it be better to add a `preferred_name` helper that is just
`full_name || slack_name` to `Person`.